content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,98 @@
+# Continuous Integration and Deployment for Machine Learning
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers for model training), model testing (validating model
+inference), and model deployment to production, as well as monitoring &
+feedback. [DVC](/) can manage data/models and reproducible pipelines, while
+[CML] can assist with orchestration, testing and monitoring.
+
+[cml]: https://cml.dev
+
+![](https://static.iterative.ai/img/ml-pipeline.png) _Basic ML pipeline_
+
+The main benefits of CI/CD (continuous integration and continuous delivery) for
```

Comment:
Title uses "Deployment". Which one are we talking about, deployment or delivery? I think the former (from the DevOps PoV). But you may want to double check with @shcheklein , @dberenbaum , or other ML folk.